src/Microsoft.AspNetCore.BuildTools.ApiCheck/Program.cs

```diff
@@ -186,66 +201,120 @@ public static int Main(string[] args)
                 oldApiListingFilters.Add(ApiListingFilters.IsInInternalNamespace);
             }
 
-            var oldApiListing = ApiListingGenerator.LoadFrom(File.ReadAllText(apiListingPathOption.Value()), oldApiListingFilters);
+            var oldApiListing = ApiListingGenerator.LoadFrom(File.ReadAllText(apiListingPathOption.Value()),
+                oldApiListingFilters);
 
             var generator = new ApiListingGenerator(assembly, newApiListingFilters);
             var newApiListing = generator.GenerateApiListing();
-            var exclusions = !exclusionsPathOption.HasValue() ?
-                Enumerable.Empty<ApiChangeExclusion>() :
-                JsonConvert.DeserializeObject<IEnumerable<ApiChangeExclusion>>(File.ReadAllText(exclusionsPathOption.Value()));
+            var knownBreakingChanges = (breakingChangesPathOption.HasValue()
+                                           ? JsonConvert.DeserializeObject<IList<BreakingChange>>(
+                                               File.ReadAllText(breakingChangesPathOption.Value()))
+                                           : null) ?? new List<BreakingChange>();
+
+            var comparer = new ApiListingComparer(oldApiListing, newApiListing);
+
+            var breakingChanges = comparer.GetDifferences();
+            var newBreakingChanges = breakingChanges.Except(knownBreakingChanges).ToList();
+            var incorrectBreakingChanges = knownBreakingChanges.Except(breakingChanges).ToList();
 
-            var comparer = new ApiListingComparer(oldApiListing, newApiListing, exclusions.ToList());
+            const string indent = "    ";
 
-            var result = comparer.GetDifferences();
-            var differences = result.BreakingChanges;
+            var breakingChangesToPrint = new List<BreakingChange>();
 
-            var compactOutput = compactOutputOption.HasValue();
-            if (!compactOutput)
+            if (newBreakingChanges.Count > 0)
             {
+                Console.WriteLine(
+                    $"ERROR: Verifying breaking changes for framework {framework.Value()} failed.");
+            }
+
+            var removedTypes = newBreakingChanges.Where(b => b.MemberId == null).OrderBy(b => b.TypeId).ToList();
+            if (removedTypes.Count > 0)
+            {
+                Console.WriteLine();
+                Console.WriteLine("The following types/interfaces have been removed.");
```

Comment:
BTW `types/interfaces` is redundant. All interfaces are types. Can clear this up in other messages below as well.